torch/nn/parameter.py

```diff
@@ -8,9 +8,12 @@
 class _ParameterMeta(torch._C._TensorMeta):
     # Make `isinstance(t, Parameter)` return True for custom tensor instances that have the _is_param flag.
     def __instancecheck__(self, instance):
-        return super().__instancecheck__(instance) or (
-            isinstance(instance, torch.Tensor) and getattr(instance, "_is_param", False)
-        )
+        if self is Parameter:
+            if isinstance(instance, torch.Tensor) and getattr(
```

Comment:
nit can be one if statement, also can technically be one boolean although this might be marginally more readable.

Reply:
yeah, I know this can be golfed but this seemed like good code flow to me